instagram/instagram/src/component/Comments.js

```diff
@@ -1,18 +1,28 @@
-import React from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 
 
-const Comments = props => {
-   
-    
-    const {username, text}=props.comment 
 
-    return (
-    <div className="commentWrapper">
-      <div className="commentName">{username}</div>
-      <div className="commentText">{text}</div>
-    </div>
-  );
+
+
+class Comments extends Component{
+
+  state={
+    comment : this.props.comment
```

Comment:
If you're not modifying the comments here, it would be better to leave comment on props and access it using `this.props.comment` in your render method